diesel/src/pg/expression/helper_types.rs

```diff
@@ -64,6 +64,11 @@ pub type RangeContains<Lhs, Rhs> = Grouped<
 #[cfg(feature = "postgres_backend")]
 pub type ContainsRange<Lhs, Rhs> = Contains<Lhs, Rhs>;
 
+/// The return type of [`lhs.is_contained_by(rhs)`](super::expression_methods::PgRangeExpressionMethods::is_contained_by)
+#[cfg(feature = "postgres_backend")]
+pub type IsContainedByRange<Lhs, Rhs> =
```

Comment:
But this should be named `RangeIsContainedBy` though.

Reply:
Ideally this type name should exactly match (in CamelCase) the name of the method, so that `#[auto_type]` works. That means I would rather prefer naming the method `range_is_contained_by` or something like that.

Reply:
Would we be able to use the helper type of IsContainedBy that is also being used by by the array method just below this type so that method names remain consistent? Or do we want to just have it be the range_is_contained_by method name and helper type

Reply:
Yes that would be possible and preferable in this case (and any other case where the type definition matches). That would need an update of the documentation down below there to mention the range methods as well.